linode/manager
# upcoming: [M3-7874] - Linode Create Refactor - Marketplace - Part 1

## Description 📝

Begins building out the Marketplace section of the new Linode Create flow 🏗️ 

> [!note]
> I'm going to trial to do small consumable PRs for the remainder of the Linode Create refactor. If these PRs seem too small, just let me know.

## Preview 📷

![Screenshot 2024-04-23 at 1 53 45 PM](https://github.com/linode/manager/assets/115251059/4e1b6e68-2751-4582-ab1e-2bfc15a8a804)

## How to test 🧪

### Prerequisites
- Turn on the `Linode Create v2` feature flag

### Verification steps
- Check http://localhost:3000/linodes/create?type=One-Click for parity with the existing Linode Create flow with what is build so far

## As an Author I have considered 🤔
